runtime/amd64.S

```diff
@@ -447,6 +447,105 @@ CFI_STARTPROC
 CFI_ENDPROC
 ENDFUNCTION(G(caml_allocN))
 
+/* Polling */
+FUNCTION(G(caml_call_poll))
```

Comment:
We should factor out common code between `caml_call_gc` and `caml_call_poll`. Multicore's amd64.S does this IINM.